### PR TITLE
Fix broken paths and results in old fine-tuning experiments

### PR DESCRIPTION
**Pull request by lintool**
These are from the "Neural Hype" paper all the way back from 2019.

### CODE REVIEW

File: src/main/resources/fine_tuning/models.yaml [hunk 1]
```diff
@@ -136,16 +136,16 @@ models:
       robust04:
         map:
           best_avg: 0.3094
-          oracles_per_topic: 0.3742
-          2-fold: 0.3043
-          5-fold: 0.3021
+          oracles_per_topic: 0.3744
```
Comment (lukuang) [inline src/main/resources/fine_tuning/models.yaml:139]: Why the numbers change, especially the oracle one?

**Comment by lintool** [comment]
These were the commands I ran:

```
# bm25+rm3
rm -r fine_tuning_results
nohup python src/main/python/fine_tuning/run_batch.py --collection robust04 --model bm25+rm3 --threads 44 --run >& logs/fine-tuning.bm25+rm3 &
python src/main/python/fine_tuning/run_batch.py --collection robust04 --model bm25+rm3 --threads 44 --run --fold_settings src/main/resources/fine_tuning/robust04-paper1-folds.json --verbose
python src/main/python/fine_tuning/run_batch.py --collection robust04 --model bm25+rm3 --threads 44 --run --fold_settings src/main/resources/fine_tuning/robust04-paper2-folds.json --verbose
# ls fine_tuning_results/robust04/run_files/ | wc -l -> 3120

# bm25+axiom
rm -r fine_tuning_results
nohup python src/main/python/fine_tuning/run_batch.py --collection robust04 --model bm25+axiom --threads 44 --run >& logs/fine-tuning.bm25+ax &
python src/main/python/fine_tuning/run_batch.py --collection robust04 --model bm25+axiom --threads 44 --run --fold_settings src/main/resources/fine_tuning/robust04-paper1-folds.json --verbose
python src/main/python/fine_tuning/run_batch.py --collection robust04 --model bm25+axiom --threads 44 --run --fold_settings src/main/resources/fine_tuning/robust04-paper2-folds.json --verbose
# ls fine_tuning_results/robust04/run_files/ | wc -l -> 360

# bm25
rm -r fine_tuning_results
nohup python src/main/python/fine_tuning/run_batch.py --collection robust04 --model bm25 --threads 44 --run >& logs/fine-tuning.bm25 &
python src/main/python/fine_tuning/run_batch.py --collection robust04 --model bm25 --threads 44 --run --fold_settings src/main/resources/fine_tuning/robust04-paper1-folds.json --verbose
python src/main/python/fine_tuning/run_batch.py --collection robust04 --model bm25 --threads 44 --run --fold_settings src/main/resources/fine_tuning/robust04-paper2-folds.json --verbose
# ls fine_tuning_results/robust04/run_files/ | wc -l -> 400

# ql
rm -r fine_tuning_results
nohup python src/main/python/fine_tuning/run_batch.py --collection robust04 --model ql --threads 44 --run >& logs/fine-tuning.ql &
python src/main/python/fine_tuning/run_batch.py --collection robust04 --model ql --threads 44 --run --fold_settings src/main/resources/fine_tuning/robust04-paper1-folds.json --verbose
python src/main/python/fine_tuning/run_batch.py --collection robust04 --model ql --threads 44 --run --fold_settings src/main/resources/fine_tuning/robust04-paper2-folds.json --verbose
# ls fine_tuning_results/robust04/run_files/ | wc -l -> 101
```

**Comment by lintool** [comment]
The last time I successfully ran the regressions was at: https://github.com/castorini/anserini/commit/e475cc4e885f5fbb223533d75a2da7abaa618c6c

I believe the changes in effectiveness come from this patch: https://github.com/castorini/anserini/pull/1975

A history of breaking changes is presented on the page:
https://github.com/castorini/anserini/blob/master/docs/experiments-forum2018.md

After this patch gets merged in I will update accordingly to note this change.

**Review by lukuang — APPROVED** [review]
lgtm

**Pull request merged**